### ISSUE TITLE
[BUG] Train/Test Split in GlobalForecaster (TinyTimesMixer) lead to empty  train/test sets.

### ISSUE THREAD

**Issue by benHeid** [sktime/sktime#7829]
**Describe the bug**

Due to the splitting it happens in the tests that the train or test sets are empty.  See: https://github.com/sktime/sktime/actions/runs/13328428251/job/37227372162?pr=7759

**To Reproduce**

```python

from build.lib.build.lib.sktime.forecasting.ttm import TinyTimeMixerForecaster
from sktime.utils.estimator_checks import check_estimator

check_estimator(TinyTimeMixerForecaster, tests_to_run=["test_global_predict_quantiles"], raise_exceptions=True)```

**Expected behavior**
Should not fail. And in case it needs to fail the error message has to be better.

**Additional context**
<!--
Add any other context about the problem here.
-->

**Versions**
<details>

<!--
Please run the following code snippet and paste the output here:

from sktime import show_versions; show_versions()
-->

</details>

<!-- Thanks for contributing! -->

**Comment by Utkarshkarki** [comment]
Is there going to be any further work on this issue

**Comment by geetu040** [comment]
Following up on this issue:

I tried reproducing it across multiple scenarios, including using the absolute minimum data length (down to a single timestamp), but I could not find any case where an error is raised. I also checked out this branch to run the exact same test suite against main, and the tests are still passing locally, even though this specific test case was removed during the global forecasting deprecation work.

Insufficient data is handled in PyTorchDataset through padding of past/future values where applicable. After going through the model implementation thoroughly, I currently do not see a scenario where the resulting datasets could become empty. There were some following PRs that touched the implementation and it could be that it was fixed during that.

If this is true, we could either close the issue or improve the logging/warnings to better explain how the dataset is constructed given the selected context/prediction lengths, since the padding and truncation behavior in this setup can be somewhat complex.

CC: @fkiraly @benHeid

**Comment by fkiraly** [comment]
@geetu040, can you please report your `check_estimator` run results?
Also please compare to https://github.com/sktime/sktime/issues/10083

**Comment by geetu040** [comment]
@fkiraly

> [@geetu040](https://github.com/geetu040), can you please report your `check_estimator` run results?

All tests PASSED!

> Also please compare to [#10083](https://github.com/sktime/sktime/issues/10083)

It's green in the [CI](https://github.com/sktime/sktime/actions/runs/25882471333/job/76065822341?pr=10081), though I it's not running the full tests
